lib/assesments/assessment_2/fia2milyas/fia2milyas_view/fia2milyas_home/fia2milyas_custom_widget/fia2ilyas_homelistview.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_interns/assesments/assessment_2/fia2milyas/fia2milyas_constant/fia2milyas_color.dart';
+import 'package:flutter_interns/assesments/assessment_2/fia2milyas/fia2milyas_view/fia2milyas_custom_widget/fia2milyas_core/fia2milyas_roundedpic.dart';
+
+class FiA2MIlyasHomeListView extends StatelessWidget {
+  const FiA2MIlyasHomeListView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: MediaQuery.sizeOf(context).width,
```

Comment:
instead of using MediaQuery.sizeOf(context).width here just use double.infinity
since we aren't multiplying it here just use double.infinity it will do the same job